spec/rails_helper.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
+require 'coveralls'
+Coveralls.wear!('rails')
+
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  puts e.to_s.strip
+  exit 1
+end
+
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+  # config.filter_gems_from_backtrace("gem name")
```

Comment:
remove commented out code

Reply:
I wanted to keep this as it is very likely we will need to use this functionality once we start running tests. Keeping the comment here will make sure there are no struggles with the syntax once it is needed.